helmfile.d/0800.codefresh-runtime-env.yaml

```
@@ -0,0 +1,61 @@
+repositories:
+# Cloud Posse incubator repo of helm charts
+- name: "cloudposse-incubator"
+  url: "https://charts.cloudposse.com/incubator/"
+
+
+releases:
+
+#######################################################################################
+## Codefresh Runtime Environments                                                    ##
+#######################################################################################
+
+#
+# References:
+#   - https://codefresh-io.github.io/cli/runtime-environments/create-runtime-environments
+#   - https://github.com/cloudposse/charts/blob/master/incubator/monochart
+#   - https://kubernetes.io/docs/tasks/inject-data-application/define-command-argument-container
+#   - https://stackoverflow.com/questions/3790454/in-yaml-how-do-i-break-a-string-over-multiple-lines
+#
+- name: "codefresh-runtime-env"
```

```suggestion
- name: "codefresh-runtime"
```

With helm, short names are key.

---

Or we can call it `codefresh-re`